DataProcessing/datax-host/src/main/scala/datax/sink/BlobSinker.scala

```diff
@@ -157,6 +163,13 @@ object BlobSinker extends SinkOperatorFactory {
     if(formatConf.isDefined && !formatConf.get.equalsIgnoreCase("json"))
       throw new Error(s"Output format: ${formatConf.get} as specified in the config is not supported")
     val outputFolders = blobOutputConf.groups.map{case(k,v)=>k->KeyVaultClient.resolveSecretIfAny(v.folder)}
+    
+    val spark = SparkSessionSingleton.getInstance(ConfigManager.initSparkConf)
+    val sc = spark.sparkContext
```

Comment:
Can you move this new code for broadcasting to a separate method , it just need to take the string as arg for getting storage account name.

Reply:
done